src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.DuplicateUserException;
+import core.basesyntax.exception.InvalidAgeException;
+import core.basesyntax.exception.InvalidLoginException;
+import core.basesyntax.exception.InvalidPasswordException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService = new RegistrationServiceImpl();
+    private static final String LOGIN = "PERSON";
+    private static final String PASSWORD = "11111111";
+    private static final String SHORT_PASSWORD = "1111";
+    private static final String EMPTY_LOGIN = "";
+    private static final int AGE = 30;
+    private static final int MINOR_AGE = 10;
+    private static final int TALE_AGE = 300;
+
+    @Test
+    void nullUser_NotOk() {
+        User user = null;
+        assertThrows(NullPointerException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void userWithNullLogin_NotOk() {
+        User user = new User(null, PASSWORD, AGE);
```

Comment:
you can use the method BeforeEach for creating a default user and use it in your test cases